app/main.py

```diff
@@ -1 +1,20 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    mv, source_file, path_file = command.split()
+    if len(command.split()) == 3 and mv == "mv":
```

Comment:
If len(command.split) will be less, than 3 - it will fail in the previous line. Please, move the if expression to the beginning